ajax/libs/jqgrid/package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "jqgrid",
-  "filename": "js/jquery.jqGrid.min.js",
-  "version": "4.6.0",
```

Comment:
This is the version we currently have on the CDN, so this attribute should not be removed.